snippets/csharp/VS_Snippets_CLR_System/System.Globalization.CultureInfo/cs/Change1.cs

```diff
@@ -1,11 +1,11 @@
-﻿// <Snippet3>
-using System;
+﻿using System;
```

Comment:
we could try to use the new syntax with this one and see if that would work. I'll make a change on the PR for us to test that.